### PR TITLE
update tox calls to use run, update all pip calls to use python -m

### PR DESCRIPTION
**Pull request by pacrob**
### What was wrong?

Calling `tox` without `run` is so `v3`. `v4` likes it to always be `tox run`.  - https://tox.wiki/en/latest/upgrading.html#cli-command-compatibility

Also update all calls to `pip` to use `python -m pip` and fix `pip freeze` reference.

#### Cute Animal Picture

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Read more in the [documentation on ReadTheDocs](https://<RTD_NAME>.readthedocs.i
 ## Quickstart
 
 ```sh
-pip install <PYPI_NAME>
+python -m pip install <PYPI_NAME>
```
Comment (wolovim) [inline README.md:17]: what's the thinking behind this inclusion?

**Review by wolovim — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pacrob) [reply README.md:17]: It's recommended: 
https://snarky.ca/why-you-should-use-python-m-pip/
https://packaging.python.org/en/latest/tutorials/installing-packages/

TL;DR, it ensures that you're installing to the same instance of python that you use for running.
